pallets/frequency-tx-payment/src/capacity_stable_weights.rs

```diff
@@ -215,4 +216,15 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(3 as u64))
 			.saturating_add(T::DbWeight::get().writes(3 as u64))
 	}
+	// Storage: Msa PublicKeyToMsaId (r:1 w:0)
+	// Storage: Handles MSAIdToDisplayName (r:1 w:1)
+	// Storage: Handles CanonicalBaseHandleToSuffixIndex (r:1 w:1)
+	// Storage: Handles CanonicalBaseHandleAndSuffixToMSAId (r:0 w:1)
+	fn change_handle(b: u32) -> Weight {
```

Comment:
I think this needs to be updated from the benchmark run. They had a different number of db writes.

Reply:
This is the handles benchmarks not the f-tx-payment run. It is generated just by copying over the details to stable weights. (Or at least I believe so)

Reply:
Fixed